applications/vanilla/controllers/api/DiscussionsApiController.php

```diff
@@ -222,6 +227,14 @@ public function index(array $query) {
 
         $in = $this->schema([
             'categoryID:i?' => 'Filter by a category.',
+            'pinned:b|n' => [
```

Comment:
As above. `b|n` does not make sense. This is a filter so `isPinned:b?` would probably make more sense.